### PR TITLE
[Icons]Revision of TechDraw Icons(blue)

### PR DESCRIPTION
**Pull request by Reqrefusion**
#14145

### CODE REVIEW

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_LeaderLine.svg
Comment (kadet1090) [inline src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_LeaderLine.svg:1]: @WandererFan Is this icon ok? This is quite a big change.

File: src/Mod/TechDraw/Gui/Resources/icons/TechDraw_ExtensionLineParallel.svg
Comment (kadet1090) [inline src/Mod/TechDraw/Gui/Resources/icons/TechDraw_ExtensionLineParallel.svg:1]: Here line that was dashed out is now presented as solid. Visually it looks OK and is readable in the UI, but I'm not sure if functionally it is fine - @WandererFan your opinion here would be very useful

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_Line2Points.svg
Comment (kadet1090) [inline src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_Line2Points.svg:1]: Please make sure that vertices are presented consistently with PR about vertex icons. I don't think that changing colors here is OK - white is used in the sketcher for presenting lines, probably would be good to stay consistent. It also won't be visible on dark themes.

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_Midpoints.svg
Comment (kadet1090) [inline src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_Midpoints.svg:1]: Concept is fine, but the same points like for `Line2Points.svg` applies - line could be white and should be a bit thicker. Please reuse line style that can be found in sketcher.

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_Quadrants.svg
Comment (kadet1090) [inline src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_Quadrants.svg:1]: Same thing - please stay with the line style from sketcher (which was used in previous icon) and ensure that vertices are presented always the same (design them once and copy)

**Review by kadet1090 — CHANGES_REQUESTED** [review]

**Review by kadet1090 — APPROVED** [review]

**Review by maxwxyz — APPROVED** [review]

**Pull request merged**